src/server/Makefile

```diff
@@ -22,6 +22,9 @@ ${OBJ_DIR}/rest_handlers.o: rest_handlers.cpp rest_handlers.h ${OBJ_DIR}/rest.o
 
```

Comment:
We should also make an issue to clean up the Makefile, as per @gemarcano's suggestion.